SECURITY.md

```diff
@@ -0,0 +1,33 @@
+# Security Policy
+
+This document outlines security procedures and general policies for this OCPP project.
+
+## Supported Versions
+
+The currently supported versions of this OCPP project are:
+
+| Version  | Supported          |
+|----------| ------------------ |
+| 2.0.0    | :white_check_mark: |
+| 0.26.0   | :white_check_mark: |
+| < 0.26.0 | :x:                |
+
```

Comment:
@jerome-benoit  actually, maybe there has been a release? I see releases listed in the [changelog](https://github.com/mobilityhouse/ocpp/blob/master/CHANGELOG.md) and there are tags listed, although none of them are listed in github as a release. and pypi has ocpp 1.0.

Reply:
but [github's releases](https://github.com/mobilityhouse/ocpp/releases) only have 0.26.0 as the last official release, although there is a 1.0.0-rc?

Reply:
Given the current state of affair, an automated release process syncing deterministically future versions between GitHub and pypi has higher priority than the others comment I've done on that file in the other PR.

Reply:
@jerome-benoit just submitted a new PR #688 which changes the release workflow process to use the same version in pyproject.toml to publish to pypi, create a git tag and create a github release. Let me know if this addresses your concern?